src/stores/Binding/Store.ts

```diff
@@ -418,6 +418,11 @@ const getInitialState = (
             ({ meta }) => meta.collectionName
         ),
 
+    getEnabledCollections: () =>
+        Object.values(get().resourceConfigs)
+            .filter(({ meta }) => !meta.disable)
+            .map(({ meta }) => meta.collectionName),
```

Comment:
We are not really using the `collectionName` mapping part - but left it in so it matches the `getCollections` up above.